.travis.yml

```diff
@@ -6,6 +6,10 @@ language: clojure
 lein: lein2
 script: JVM_OPTS="-Djava.library.path=$PWD/hadoop-lzo-native/lib" lein2 sub with-profile dev,provided test
 before_install:
+    - cat /etc/hosts # optionally check the content *before*
```

Comment:
https://github.com/travis-ci/travis-ci/issues/5227#issuecomment-165131913
Buffer overflow in Java_java_net_Inet4AddressImpl_getLocalHostName of OpenJDK6 and OpenJDK 7